upbound_uxp/assets/dashboards/upbound_uxp_min_set.json

```diff
@@ -0,0 +1 @@
k Duration Seconds Count","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:uxp.workqueue_work_duration_seconds_bucket.count{*} by {host,pod,name,le}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":8,"width":4,"height":2}},{"id":6712236079650739,"definition":{"title":"Workqueue Work Duration Seconds Sum","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:uxp.workqueue_work_duration_seconds_sum{*} by {host,pod,name}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":8,"width":4,"height":2}},{"id":3667451147019795,"definition":{"title":"Upjet Terraform Active CLI Invocations","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.upjet_terraform_active_cli_invocations{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":10,"width":4,"height":2}},{"id":4282148986424876,"definition":{"title":"Upjet Terraform Running Processes","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.upjet_terraform_running_processes{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":10,"width":4,"height":2}},{"id":2395363743244600,"definition":{"title":"Controller Runtime Reconcile Errors","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.controller_runtime_reconcile_errors_total{*}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":10,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
Can you please add a logo to this dashboard, as well as any helpful text or links to documentation?

Additionally, can you double check the format of the description? It looks like it's using a lot of headers:
<img width="559" alt="Screenshot 2023-06-23 at 4 14 56 PM" src="https://github.com/DataDog/integrations-extras/assets/22504341/9e1d3257-1006-4434-b4bc-6cd27b3cbfaf">